prov/efa/src/efa_base_ep.h

```diff
@@ -39,6 +39,28 @@
 #define efa_rx_flags(efa_base_ep) ((efa_base_ep)->util_ep.rx_op_flags)
 #define efa_tx_flags(efa_base_ep) ((efa_base_ep)->util_ep.tx_op_flags)
 
+#define EFA_PROV_NAME "efa"
+#define EFA_DIRECT_PROV_NAME "efa-direct"
+
+/* TODO - CHECK THESE */
+#define EFA_EP_TYPE_IS_RDM(_info) \
+	(_info && _info->ep_attr && (_info->ep_attr->type == FI_EP_RDM) && !strcasecmp(_info->fabric_attr->name, EFA_PROV_NAME))
```

Comment:
Both efa and efa-direct has rdm ep type. Making this macro for efa only seems goofy to me.

Reply:
Changed to `EFA_INFO_TYPE_IS_RDM`